kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -619,29 +618,29 @@ def run_sanlock_checker():
 
 
     def read_fencer_heartbeat(self, host_uuid, vg_uuid):
-        current_read_heartbeat_time = [None]
-        current_vm_uuids = [None]
+        current_read_heartbeat_time = None
```

Ruff (0.12.2)

Local variable `current_read_heartbeat_time` is assigned to but never used

Remove assignment to unused variable `current_read_heartbeat_time`

(F841)